lib/bas.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 require_relative "bas/version"
+require_relative "bas/bot/base"
+require_relative "bas/shared_storage/base"
+require_relative "bas/shared_storage/default"
+require_relative "bas/shared_storage/postgres"
```

Comment:
I don’t think this should be added by default when the gem is imported. Users should have the flexibility to import only the shared storage implementation required for their specific use case. Currently, postgres is the only option available, but in the future, we plan to add more implementations. For example, if a user wants to use shared storage with S3, importing the postgres module should not be enforced.

Let me know what you think! :thinking: